src/UWP/ArcGISRuntime.UWP.Samples/Samples/Layers/KmlNetworkLinks/KmlNetworkLinks.xaml.cs

```diff
@@ -0,0 +1,133 @@
+﻿// Copyright 2017 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using Esri.ArcGISRuntime.Geometry;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Ogc;
+using Esri.ArcGISRuntime.Portal;
+using System;
+using System.Collections.Generic;
+using Windows.UI.Core;
+using Windows.UI.Popups;
+
+namespace ArcGISRuntime.UWP.Samples.KmlNetworkLinks
+{
+	public partial class KmlNetworkLinks
+    {
+        // Member (aka global) variable to hold a reference to the KML dataset
+        KmlDataset _myKmlDataset;
+
+        public KmlNetworkLinks()
+		{
+			InitializeComponent();
+
+			// Initialize the map
+			Initialize();
+		}
+
+		private async void Initialize()
+		{
+            try
+            {
+                // Set up the map with a basemap
+                MyMap.Basemap = Basemap.CreateDarkGrayCanvasVector();
+
+                // First clear the operational layers
+                MyMap.OperationalLayers.Clear();
```

Comment:
Is this necessary at this point? The OperationalLayers should be empty by default when first loading the sample.